mod/deckorate_attribution/assets/script/clipboard.js.coffee

```diff
@@ -1,9 +1,10 @@
 $ ->
   $("body").on "click", ".copy-button", ->
-    clipboardContent = $("#clipboard").text()
-    
-    navigator.clipboard.writeText(clipboardContent)
+    content = $(this).closest(".tab-pane.active").find("._clipboard").html()
+    blob = new Blob([content], { type: "text/html" });
+    richTextInput = new ClipboardItem({ "text/html": blob });
+    navigator.clipboard.write([richTextInput])
       .then ->
-        console.log "Text copied to clipboard: #{clipboardContent}"
+        console.log "Text copied to clipboard: #{content}"
       .catch (error) ->
         console.error "Copy to clipboard failed:", error
```

Comment:
`console.log` is for debugging and generally shouldn't be in production code. Perhaps we can comment this out?